test/integration/render-tests/text-pitch-alignment/viewport-text-depthtest/style.json

```diff
@@ -2,8 +2,7 @@
   "version": 8,
   "metadata": {
     "test": {
-      "height": 256,
-      "allowed": 0.00025
+      "height": 256
```

Comment:
This test was incorrectly reverted in https://github.com/mapbox/mapbox-gl-js/pull/10562 and was passing due to a high "allowed." By removing the allowed and correcting the test we should now correctly catch a regression to order.